webapp/api_handlers.go

```diff
@@ -18,6 +20,7 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/appengine"
 	"google.golang.org/appengine/datastore"
+	"google.golang.org/appengine/urlfetch"
```

Comment:
FWIW, if we decide to go ahead with #107 , we won't be using `urlfetch` anymore. `urlfetch` isn't used extensively in this PR yet, but I still suggest creating a wrapper for `urlfetch` so that we can switch to generic network I/O later.

Reply:
Will save that pain for another day.